templates/zerver/help/zulip-on-windows.md

```diff
@@ -0,0 +1,44 @@
+# Using Zulip on Windows
+
+Zulip has a Windows client, compatible with Windows versions 7, 8 and 10 and is identical in
+functionality to the Zulip Web Application. The client
+can be download through https://zulip.org/.
+
+The client is regularly updated and is a wrapper of the Web App, but with enhanced
+notifications and can run in the background. The current version does not contain
+any significant features additional to those in the Web App.
+
+!!! warn ""
+    **Note:** Zulip is no longer running on a browser, many external keyboard shortcuts
```

Comment:
@540KJ This still hasn't been fixed. Please note the semicolon that replaces the comma